util/src/main/java/tech/pegasys/teku/util/config/ConstantsReader.java

```diff
@@ -15,18 +15,55 @@
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
+import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.io.InputStream;
 import java.lang.reflect.Field;
 import java.lang.reflect.Modifier;
+import java.util.List;
 import java.util.Map;
+import java.util.Optional;
 import java.util.function.Function;
+import java.util.stream.Collectors;
 import org.apache.tuweni.bytes.Bytes;
+import tech.pegasys.teku.infrastructure.io.resource.ResourceLoader;
 import tech.pegasys.teku.infrastructure.unsigned.UInt64;
 import tech.pegasys.teku.ssz.type.Bytes4;
 
 class ConstantsReader {
```

Comment:
Not for this PR, but I'm beginning to wonder if we should change this to just explicitly set the values in `Constants` from a given `SpecConfig`.  We know there won't be any new fields added to this class so the magic reflection isn't needed and then we could avoid having duplicate code for reading the config files.